src/broker/broker_monitor.c

```diff
@@ -519,9 +505,18 @@ main (int argc, char **argv)
 	}
 //  FillConsoleOutputCharacter(h_console, ' ', scr_info.dwSize.X * scr_info.dwSize.Y, top_left_pos, &size);
 #else
-      win = initscr ();
-      timeout (refresh_sec * 1000);
-      noecho ();
+      snprintf (cmd, PATH_MAX, "%s -t -n %d", WATCH_CMD, refresh_sec);
```

Comment:
if there is not a watch command in the running machine, you need to output a message such as "please install watch command for using this option..".